charts/loki-distributed/Chart.yaml

```
@@ -3,7 +3,7 @@ name: loki-distributed
 description: Helm chart for Grafana Loki in microservices mode
 type: application
 appVersion: 2.1.0
-version: 0.22.1
+version: 0.22.2
```

As mentioned, please bump the minor version. I consider this a new feature, not a patch.
```suggestion
version: 0.23.0
```